store-and-retrieve-data/use-storagehub-sdk/get-started.md

```diff
@@ -1,4 +1,105 @@
 ---
 title: Get Started
 description: Get up and running quickly with the StorageHub SDK.
```

Comment:
```suggestion
description: TODO
```
What does up and running mean? Am I connecting to a network? Sending some kind of request? This title doesn't tell me and the page currently doesn't answer this either.